cluster/common.sh

```diff
@@ -376,7 +376,8 @@ function generate-etcd-cert() {
                 "usages": [
                     "signing",
                     "key encipherment",
-                    "server auth"
+                    "server auth",
+                    "client auth"
```

Comment:
why isn't the client profile sufficient?

Reply:
I used mutual TLS so I need to generate both server cert and client cert. Then I added client auth here because of a know issue in etcd https://github.com/etcd-io/etcd/issues/9785#issuecomment-396715692.

Reply:
The dual use of etcd certificates as serving and client certs is unfortunate, but known